runtime/app/xwalk_main_delegate.cc

```diff
@@ -76,6 +77,10 @@ void XWalkMainDelegate::PreSandboxStartup() {
       command_line->GetSwitchValueASCII(switches::kProcessType);
   InitLogging(process_type);
 #endif
+
+#if !defined(OS_ANDROID)
```

Comment:
@rakuco Please refer to, https://github.com/crosswalk-project/crosswalk/pull/3788